intellij-extension/about.hbs.md

```diff
@@ -39,6 +39,8 @@ This extension gives the following features.
   For more information about a typical monorepo setup, see
   [Working with microservices in a monorepo](using-the-extension.hbs.md#mono-repo).
 
+-  **Note** The new variation of the out-of-the-box (OOTB) Basic supply chains, which output [Carvel packages](../scc/carvel-package-supply-chain.hbs.md) to enable configuring multiple runtime environment, is not yet supported in this plugin.
```

Comment:
Could you explicitly state the 2 new SC names? As a reader of this extension when I choose a SC, it'll be helpful to see the two ones I should avoid without having to click through and learn about them.

Reply:
@Petahhh updated